pkg/apis/core/validation/validation.go

```diff
@@ -3382,6 +3365,10 @@ func ValidatePodSecurityContext(securityContext *core.PodSecurityContext, spec *
 				allErrs = append(allErrs, field.Invalid(fldPath.Child("shareProcessNamespace"), *securityContext.ShareProcessNamespace, "ShareProcessNamespace and HostPID cannot both be enabled"))
 			}
 		}
+
+		if securityContext.Sysctls != nil {
+			allErrs = append(allErrs, validateSysctls(securityContext.Sysctls, fldPath.Child("sysctls"))...)
```

Comment:
this is our only chance to make validateSysctls as tight as we want it to be. should the following things be done?
* require name
* require value (not currently done. is it ever valid to set a sysctl to an empty value?)
* prevent duplicate names (not currently done)
* validate name format and length
* validate value format and length (not currently done. are there documented limits on sysctl values and length?)